### PR TITLE
Add manuneo layout

### PR DESCRIPTION
**Pull request by mbarkhau**
This generates keymap.c files based on a markdown file.
I couldn't get the unicode input to work so I hacked together something based on macros.

### CODE REVIEW

**Comment by ezuk** [comment]
This is awesome! A couple of quick questions:

1) Would you mind including your compiled hex file as well?

2) Any chance you could write up a quick readme, mainly covering what needs to be installed for the Python script to work? Just a quick rundown of what people can do to grab it and create their own keymaps?

**Comment by mbarkhau** [comment]
Yes, I'll be happy to clean it up. Can you point me to some help on how to get UC/UNICODE(0x1234) working?

**Comment by ezuk** [comment]
How's this thread? https://github.com/jackhumbert/qmk_firmware/issues/86

**Comment by mbarkhau** [comment]
Yes that looks helpful, I'll see if I can use [keymap_unicode.c](https://github.com/jackhumbert/qmk_firmware/blob/master/quantum/keymap_unicode.c) instead of the macro hack.

**Comment by mbarkhau** [comment]
Is there a way to invoke the `action_function` apart from using a macro?

**Comment by ezuk** [comment]
I think the answer to that is no, but am not sure. @jackhumbert can you confirm?

File: keyboard/ergodox_ez/keymaps/german-manuneo/keymap.c [hunk 1]
```diff
@@ -0,0 +1,783 @@
...
+ * |MO(4)| X  | P  | F  | W  | G  |HOME |TG(2)| H  | J  | K  | L  | Q  |  Z  |
+ * !-----+----+----+----x----x----!     !     !----x----x----+----+----+-----!
+ * |MO(1)| U  | I  | A  | E  | O  |-----!-----! S  | N  | R  | T  | D  | SS  |
+ * !-----+----+----+----x----x----!     !     !----x----x----+----+----+-----!
+ * |MO(3)| UE | OE | AE | C  | V  |END  | TAB | B  | M  |COMM| DOT| UP |  Y  |
+ * '-----+----+----+----+----+----------'----------+----+----+----+----+-----'
+ *  |    |    |LGUI|LALT|LCTL|                     !RCTL|RALT|LEFT|DOWN|RGHT|
+ *  '------------------------'                     '------------------------'
+ *                         .-----------. .-----------.
+ *                         |INS  |TG(2)| !M(UM)|DELT |
```
Comment (ezuk) [inline keyboard/ergodox_ez/keymaps/german-manuneo/keymap.c]: Just a quick q about this: What's `M(UM)`? Want to mention this in the readme?

**Comment by ezuk** [comment]
@mbarkhau -  I am merging this now because I love it :) I still have a question on it, on the layout itself, re M(UM).

**Comment by mbarkhau** [comment]
The UM macro is the unicode mode switch. It cycles through the unicode entry modes for windows and linux (MacOS not tested).

**Comment by ezuk** [comment]
Nice! I want you to know I'm already using your script on another branch, and loving it. One thing I would add here is comment-style "placeholders" for the empty keys in the generated layout, like the ones you [can see here](https://github.com/romanzolotarev/qmk_firmware/blob/a429374b7842cbacb68e11c5b68b868790823c2a/keyboard/ergodox_ez/keymaps/romanzolotarev-norman-plover-osx/keymap.c).

Just a thought. The script is super handy!

**Comment by mbarkhau** [comment]
Sure thing, here you go: https://github.com/jackhumbert/qmk_firmware/pull/235

**Pull request merged**